src/dashboard/src/components/administration/models.py

```diff
@@ -50,3 +51,4 @@ class ArchivesSpaceConfig(models.Model):
     access_conditions = models.CharField(max_length=50, blank=True, verbose_name='Conditions governing access')
     use_conditions = models.CharField(max_length=50, blank=True, verbose_name='Conditions governing use')
     repository = models.IntegerField(default=2, verbose_name='ArchivesSpace repository number')
+    inherit_notes = models.BooleanField(default=True, verbose_name='Inherit digital object notes from the parent component')
```

Comment:
But those changes were for a particular client, and I think we wanted to move away from the somewhat unusual behavior by having it default to not inheriting, and the client can set it to True for their usecase.